tests/scr-support_scrolled-doc/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en" lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<script>
+			/**
+			 * A simple change that proves that scripting is enabled.
+			 * 
+			 */ 
+			function test() {
+			  const div_no_scripting = document.getElementById('no_scripting');
+			  div_no_scripting.style.display = 'none';
+			  const div_scripting = document.getElementById('scripting');
+			  div_scripting.style.display = 'block';
+			}
+			window.addEventListener('load', test);
+		</script>
+		<title>rendition:flow set to scrolled-doc when scripting is supported</title>
+	</head>
+	<body>
+		<section id="flow-015">
+			<h2>Scrolled Flow</h2>
+			<div id="no_scripting" style="display: block">
+				<p>This reading system does not support scripting. The test is not relevant, and can be marked as "null" in the report</p>
+			</div>
+			<div id="scripting" style="display: none">
+				<p>
+					Tests whether setting the global <code>rendition:flow</code> property to <code>scrolled-continuous</code> results in
```

Comment:
```suggestion
					Tests whether setting the global <code>rendition:flow</code> property to <code>scrolled-doc</code> results in
```

Reply:
As I said, I messed up the two descriptions. Settled now.